gensim/corpora/lowcorpus.py

```diff
@@ -166,8 +167,8 @@ def save_corpus(fname, corpus, id2word=None, metadata=False):
 
         if truncated:
             logger.warning("List-of-words format can only save vectors with "
-                            "integer elements; %i float entries were truncated to integer value" %
-                            truncated)
+                           "integer elements; %i float entries were truncated to integer value" %
```

Comment:
We want no vertical indent in gensim. I don't know if you're using some tool or making the changes manually in this PR, but can you fix those cases too (use hanging indent)?